pytorch_lightning/utilities/imports.py

```diff
@@ -129,6 +130,12 @@ def _compare_version(package: str, op: Callable, version: str, use_base_version:
 else:
     _IPU_AVAILABLE = False
 
+from habana_frameworks.torch.utils.library_loader import is_habana_avaialble, load_habana_module
+
+_HPU_AVAILABLE = is_habana_avaialble()
+if _HPU_AVAILABLE:
+    load_habana_module()
+    os.environ["PL_TORCH_DISTRIBUTED_BACKEND"] = "hccl"
```

Comment:
I think we could add this logic in the accelerator setup method

Reply:
I want even simple tensor creation on device to succeed. Will it work if we move it to accelerator setup ?